homeassistant/components/tractive/device_tracker.py

```diff
@@ -0,0 +1,162 @@
+"""Support for Tractive device trackers."""
+
+import asyncio
+import logging
+
+from homeassistant.components.device_tracker import SOURCE_TYPE_GPS
+from homeassistant.components.device_tracker.config_entry import TrackerEntity
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from .const import (
+    DOMAIN,
+    SERVER_UNAVAILABLE,
+    TRACKER_HARDWARE_STATUS_UPDATED,
+    TRACKER_POSITION_UPDATED,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up Tractive device trackers."""
+    client = hass.data[DOMAIN][entry.entry_id]
+
+    trackables = await client.trackable_objects()
+
+    entities = await asyncio.gather(
```

Comment:
How much data is this requesting from the tractive server?
Each call to `create_trackable_entity ` also triggers 4 requests to tractive?
Would it be better for tractive if we no do all these in parallel?

Reply:
Not so much, matter of 1-2kb for all requests

Reply:
I guess it's ok.
Since I also work in a company with an open API, I see some integrations that generate a lot of unnecessary loads, I always try to think of that :)

Reply:
Anyway, the requests are only made on the startup, if the user won't restart HA 1000 times in row, this traffic won't be noticeable. Also I don't think there are lots of HA users who are tractive clients.

Reply:
@Danielhiversen BTW, would you like to join me as a maintainer?

Reply:
Sure